City-of-Helsinki/berth-reservations-ui
# VEN-1221 | Redirect the user to login after step 2

## Description :sparkles:
- Make all form tabs private routes
- Refactor the selection page to use the same layout for both berth and winter storage applications
- Remove the switch application fields from the selection page

## Issues :bug:

### Closes :no_good_woman:
[**VEN-1221**](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1221): Redirect the user to the login view in all applications

### Related :handshake:
[VEN-1404](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1404): Transfer application: Change the current berth view to the customer info view

## Testing :alembic:

### Automated tests :gear:️

### Manual testing :construction_worker_man:
- Go to the second step (selected harbors or winter storage areas) without logging in
- The selection page no longer shows a note whether this is a new application or a transfer application
- On the same page you would see a note about the authentication
- The next button redirects the user to the login page
- After logging in, the user should be back to the third step of the form (with the same language option selected)
- Trying to access any of the routes manually (e.g. by typing the route to the third step in the browser) without logging in, should redirect the user to the login page.
- If the user repeats the same process but with logging in first, the second step doesn't show the authentication note, and the next button leads directly to the third step.

## Screenshots :camera_flash:
![image](https://user-images.githubusercontent.com/23040926/139820246-00bc6576-5b45-446e-9324-5d09a92e9bc7.png)

## Additional notes :spiral_notepad:
- Switch application fields to be added to the application info view in another PR
